src/components/macro/macro-action-editor/tab/macro-key/macro-key.component.ts

```diff
@@ -0,0 +1,73 @@
+import { KeyAction } from '../../../../../config-serializer/config-items/key-action/KeyAction';
+import { EditableMacroAction } from '../../../../../config-serializer/config-items/macro-action/EditableMacroAction';
+import { MacroSubAction } from '../../../../../config-serializer/config-items/macro-action/MacroAction';
+import { KeypressTabComponent } from '../../../../popover/tab/keypress';
+import { Tab } from '../../../../popover/tab/tab';
+import { Component, Input, OnInit, ViewChild } from '@angular/core';
```

Comment:
True, but if we make groups, then tslint test passes. Feel free to create an issue about the import orders as I mentioned before, I admit, that the current ordering is not the best, but I want to avoid random orders.